ScanbotSDKexample/build.gradle

```diff
@@ -24,6 +34,10 @@ allprojects {
         maven {
             url 'http://nexus.scanbot.io/nexus/content/repositories/snapshots/'
         }
+        maven {
```

Comment:
A new requirement to use the Scanbot SDK? 
If so, we should also add it in documentation.